geoportal/c2cgeoportal_geoportal/scripts/manage_users.py

```diff
@@ -82,74 +84,77 @@ def main():
     options = parser.parse_args()
     username = options.user
 
-    # Loading schema name from config and setting its value to the
-    # corresponding global variable from c2cgeoportal_geoportal
+    config_uri = get_config_uri(options)
+    loader = get_config_loader(config_uri)
+    loader.setup_logging()
+    settings = loader.get_wsgi_app_settings()
 
     # Ignores pyramid deprecation warnings
     warnings.simplefilter("ignore", DeprecationWarning)
```

Comment:
This can be in all the scripts

Reply:
This will have no effect as we do not load the app anymore. see https://github.com/Pylons/pyramid/search?q=DeprecationWarning&unscoped_q=DeprecationWarning
This only appears in routes and views.

Reply:
OK, thanks for investigation :-)